RIAssigner/data/PandasData.py

```diff
@@ -44,13 +44,17 @@ def _init_rt_column_info(self):
     def _init_ri_column_info(self):
         """ Initialize retention index column name and set its position next to the retention time column. """
         self._ri_index = 'retention_index'
-        self._ri_position = self._rt_position + 1
+        if self._ri_index in self._data.columns:
+            self._ri_position = self._data.columns.get_loc(self._ri_index)
```

Comment:
That looks nice, I've had no idea something like `get_loc` existed.

Reply:
This is only for Pandas data frames. Found it in the documentation [here](https://pandas.pydata.org/docs/reference/api/pandas.Index.get_loc.html#pandas.Index.get_loc).

Reply:
yup, I checked it out already :)